DevelopGitHub/GitHubExtension/GitHubExtension.Templates/ExtensionMethods/IEnumerableExtension.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+
+namespace GitHubExtension.Templates.ExtensionMethods
+{
+    public static class IEnumerableExtension
+    {
+        public static bool IsEmpty(this IEnumerable source)
```

Comment:
Do you really need this extension method? It's nice but needed? Can you explain use case.